lib/workflows/send_errored_vm_email.py

```diff
@@ -0,0 +1,180 @@
+from typing import List, Optional, Union
+
+from email_api.emailer import Emailer
+from openstack_query import ServerQuery, UserQuery
+
+from enums.cloud_domains import CloudDomains
+from enums.query.props import ServerProperties
+from enums.query.query_presets import QueryPresetsGeneric, QueryPresetsDateTime
+
+from structs.email.email_params import EmailParams
+from structs.email.email_template_details import EmailTemplateDetails
+from structs.email.smtp_account import SMTPAccount
+
+
+def find_servers_with_errored_vms(
+    cloud_account: str, time_variable:int, from_projects: Optional[List[str]] = None
+):
+    """
+    :param cloud_account: string represents cloud account to use
+    :param from_projects: A list of project identifiers to limit search in
```

Comment:
Missing `time_variable` in this docstring

Reply:
Docstring edited to explain relevant parameter.